tests/testthat/_snaps/cnd-entrace.md

```diff
@@ -40,7 +40,7 @@
        12.     \-rlang:::c()
        13.       +-base::withCallingHandlers(...)
        14.       +-rlang::with_abort(f())
-       15.       | \-base::withCallingHandlers(...)
+       15.       | \-base::withCallingHandlers(expr, error = `<fn>`)
```

Comment:
Again, not very important, but why does this change?

Reply:
The injected handler is now a symbol. This allows `as_label()` to keep the whole call instead of eliding.